src/abi.md

```diff
@@ -79,22 +344,46 @@ The *`link_section` attribute* specifies the section of the object file that a
 pub static VAR1: u32 = 1;
 ```
 
-## The `export_name` attribute
+r[abi.link_section.def]
+An item with the *`link_section` attribute* is placed in the specified section when linking. The section specified shall not violate the constraints on section names on the target, and shall not be invalid for the item type, no diagnostic is required.
```

Comment:
Similar to our conversation before about ndr, I think it would be best to avoid this for now until we have a decision on it.